app/src/main/java/net/gsantner/markor/model/Document.java

```diff
@@ -274,7 +281,11 @@ public synchronized boolean saveContent(final Context context, final String cont
 
         if (success) {
             setContentHash(content);
-            _modTime = lastModified();
+            final long curModTime = lastModified();
+            if (_modTime >= curModTime) {
+                Log.w("MARKOR_DOCUMENT", "File modification time unchanged after write");
```

Comment:
To aid debugging. I think we should leave this or something similar in there

Reply:
OK to me to add more logs (albeit it won't be exporable by some in-app button, which is OK).  
Please use getClass().getName for the first parameter